Readme.md

```diff
@@ -1,61 +1,36 @@
-# Subutai Social Foundation Website
+# Development
 
-This project contains the SSF Website. It uses Jekyll to generate a static
-site from markdown files. You can read more about how to create content for
-the website from the [SSF Website Project](https://confluence.subutai.io/x/F4AWAQ) documentation space in Confluence.
+This project contains the Web Site. It uses the npm package and the GULP build system. In order to run the site and connect all dependent components you need to read "Running Subutai Website"
 
-## Running the Site
+#### Required packages
 
-Clone this project to your local machine and make sure you have a hypervisor
-and vagrant installed. If you don't have one of the non-free hypervisors you can
-install VirtualBox which is free. Once you have these prerequisites installed
-you can fire up the ['subutai/basic'] vagrant box associated with this project by issueing the
-following command within the directory you cloned this project into:
+ * Node
+ * Gulp
+ * npm
 
-    vagrant up
 
-VirtualBox is the default provider for Vagrant. If you would like to use vmware
-or parallels on Mac OS X you can issue one of the following respective commands:
+## Running Subutai Website
 
-    vagrant up --provider vmware_fusion
-    vagrant up --provider vmware_desktop
-    vagrant up --provider virtualbox
-    vagrant up --provider parallels
+#### Installing Nodejs and Gulp
+```
+sudo apt-get install -y nodejs
+```
 
+or download the latest version from the official website: https://nodejs.org/
 
-Once vagrant fires up the box, you can ssh into it with *vagrant ssh*, then you must
-configure aws credentials by the following command and set access, secret keys and region:
+```
+npm install -g gulp-cli  (maybe sudo)
```

Comment:
after this step `npm install gulp -D` should be executed, without it gulp not starts